docs/source/public/contributing_inputs.rst

```diff
@@ -28,16 +28,25 @@ For each submitted directory:
 Detailed Instructions
 *********************
 
-**1. Clone the GitHub repository**
+**1. Fork the GitHub repository**
 
-You can clone this repository, meaning making a local copy of the repository, to make it easier to add new files and push changes back to the remote repository on GitHub.com.
+First, you can create a fork of the GitHub repository. You will then make the changes (adding the files) first in the forked repository and afterwards propose the changes to the upstream repository (`OpenFreeEnergy/IndustryBenchmarks2024`).
+
+* In the `OpenFreeEnergy/IndustryBenchmarks2024` repository in the top-right corner of the page, click **Fork**.
+* Under "Owner", select an owner for the forked repository (likely yourself). The repository name can stay as given, `IndustryBenchmarks2024`.
+* Click **Create Fork**.
+
+**2. Clone the forked GitHub repository**
+
+You can clone the fork created in step 1, meaning making a local copy of the repository, to make it easier to add new files and push changes back to the remote repository on GitHub.com.
 
 .. code-block:: bash
 
-   git clone https://github.com/OpenFreeEnergy/IndustryBenchmarks2024.git
+   git clone https://github.com/Owner/IndustryBenchmarks2024.git
 
+where ``Owner`` is the name of the owner that was selected in step 1.
 
-**2. Create and work off of a remote branch**
+**3. Create and work off of a remote branch**
 
 You can create a remote branch that will allow you to make changes to the repository (e.g. adding new files) without the changes immediately affecting the main reporitory.
```

Comment:
```suggestion
You can create a remote branch that will allow you to make changes to the repository (e.g. adding new files) without the changes immediately affecting the main repository.
```
This is a bit confusing now that we're working off a fork. Maybe "main repository" -> "main branch"?